doc/_admin-guide/070_Destinations/040_File/README.md

```diff
@@ -45,7 +45,7 @@ destination d_file {
 
 **NOTE:** When using this destination, update the configuration of your log
 rotation program to rotate these files. Otherwise, the log files can
-become very large.
+become very large. See [[Configuring log rotation]] for available options.
```

Comment:
Tiny enhancements, the build of the doc site can discover linkable texts like this, and will create the link automatically, you do not need the [[ ]] here